pom.xml

```diff
@@ -55,7 +55,7 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>ice4j</artifactId>
-      <version>1.0-20160310.155819-18</version>
+      <version>1.0-SNAPSHOT</version>
```

Comment:
Yes. The file has to be updated with the new snapshot version before merging (should have mentioned that in the PR comment).